src/main/java/chess/Bishop.java

```diff
@@ -1,32 +1,29 @@
 package chess;
-
-import java.util.List;
 import java.util.ArrayList;
 
 class Bishop extends Piece {
-    Bishop(final Color color, final Position source) {
+
+    private GetPossibleMovePiece getPossibleMovePiece;
+
+    Bishop(final Color color, final Position source, final ChessBoard chessBoard) {
         super.setColor(color);
         super.setSymbol(Symbol.B);
         super.setStatus(Status.ALIVE);
         super.setPosition(source);
+        super.setChessBoard(chessBoard);
+        this.getPossibleMovePiece = new GetPossibleMovePiece(this);
     }
 
-    public void kill(final Piece target) {
-    }
-
-    public void move(final Piece target) {
-    }
-
-    public void die() {
-    }
-
-    public boolean isPosibleMove(final Piece target) {
-        return true;
-    }
 
-    public List<Piece> getPosibleMoves(final ChessBoard chb) {
-        List<Piece> array = new ArrayList<Piece>();
-        return array;
+    public ArrayList<Position> getPossibleMoves(final ChessBoard chb) {
```

Comment:
The argument "ChessBoard chb" is not used in method, so it shouldn't be passed.

Reply:
Change made can you review it?